utility/test/module_test.go

```diff
@@ -63,7 +65,7 @@ func NewTestingUtilityContext(t *testing.T, height int64) utility.UtilityContext
 // TODO(andrew): Take in `t` and fail the test if there's an error
 func newTestPersistenceModule(databaseUrl string) modules.PersistenceModule {
 	cfg := modules.Config{
-		Persistence: &test_artifacts.MockPersistenceConfig{
+		Persistence: &typesPers.PersistenceConfig{
```

Comment:
Cross module import

Reply:
Maybe addressed in https://github.com/pokt-network/pocket/pull/235/commits/1924e0e7e3c6988d13cb0b4a2a4edc5c07c64f2b#diff-e31d62c553295c0f69210e9d518a19d448a13f20bb057ad8d94b58a13bfcfa05L11